setup.py

```diff
@@ -22,6 +22,13 @@
     ]
 
 
+extras_require = {
+    'tests': tests_require,
+    'deletepermission': [
+        'collective.deletepermission',
```

Comment:
If one of the items is a variable and the other one is a literal assignment it looks to me as if it is not "on the same level of abstraction". But this is just a suggestion and may be opinionated 😉 , so i'll leave it up to you if you want to change it.